common/styles/app.css

```diff
@@ -70,13 +70,6 @@ button .glyphicon-trash {
     margin: 0px 0 12px 0;
 }
 
-.entity-key{
-    /*font-weight: bold;
-    color: rgb(118, 118, 118);
-    font-size: 14px; */
-    text-transform: capitalize;
```

Comment:
Only the old record app both uses this `.entity-key` class and needs the capitalization rule. Other apps that use this class either already have their text capitalized or use a `displayname` property from ermrestjs, so they don't need the capitalization rule. I've moved this rule to the record app.